### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by notchrisarcand**
HAHA

### CODE REVIEW

**Comment by notchrisarcand** [comment]
<rubocop />Checked commit https://github.com/matoska/sandbox/commit/ee2b35fc4d193753c3020f8e36ae9415981ea13a with ruby 2.3.3, rubocop 0.47.1, and haml-lint 0.20.0
0 files checked, 0 offenses detected
Everything looks good. :+1:

**Comment by notchrisarcand** [comment]
@notchrisarcand add_label bug, duplicate

**Comment by chrisarcand** [comment]
@notchrisarcand add_label invalid

**Comment by notchrisarcand** [comment]
<rubocop />Checked commits https://github.com/matoska/sandbox/compare/ee2b35fc4d193753c3020f8e36ae9415981ea13a~...72f8a2c23e7e5b3cddbbc027a405ab7a0c9cf815 with ruby 2.3.3, rubocop 0.47.1, and haml-lint 0.20.0
0 files checked, 0 offenses detected
Everything looks good. :cookie:

**Comment by chrisarcand** [comment]
@notchrisarcand add_labels wip

**Comment by chrisarcand** [comment]
@notchrisarcand assign @chrisarcand

**Comment by chrisarcand** [comment]
@notchrisarcand remove_label duplicate

**Comment by chrisarcand** [comment]
@notchrisarcand remove_label duplicate

**Comment by chrisarcand** [comment]
@notchrisarcand remove_label duplicate

**Comment by chrisarcand** [comment]
@notchrisarcand remove_label bug

**Comment by chrisarcand** [comment]
@notchrisarcand remove_label bug

**Comment by notchrisarcand** [comment]
@chrisarcand unrecognized command 'remove_label', ignoring...

Accepted commands are:

**Comment by chrisarcand** [comment]
@notchrisarcand remove_label bug

**Comment by notchrisarcand** [comment]
<rubocop />Checked commits https://github.com/matoska/sandbox/compare/ee2b35fc4d193753c3020f8e36ae9415981ea13a~...99bb8bd6fc94e900ad9b0259c00f4296f9315f8a with ruby 2.3.3, rubocop 0.47.1, and haml-lint 0.20.0
0 files checked, 0 offenses detected
Everything looks good. :trophy:

**Comment by chrisarcand** [comment]
@notchrisarcand remove_label bug

**Comment by chrisarcand** [comment]
@notchrisarcand remove_label bug

**Comment by chrisarcand** [comment]
@notchrisarcand remove_label bug

**Comment by chrisarcand** [comment]
@notchrisarcand remove_label bug

**Comment by chrisarcand** [comment]
test

@notchrisarcand remove_label bug

**Comment by chrisarcand** [comment]
@notchrisarcand remove_label bug

**Comment by chrisarcand** [comment]
@notchrisarcand remove_label bug

**Comment by chrisarcand** [comment]
@notchrisarcand remove_label bug

**Comment by chrisarcand** [comment]
@notchrisarcand assign notchrisarcand

**Comment by chrisarcand** [comment]
@notchrisarcand rm_label wip

**Comment by chrisarcand** [comment]
@notchrisarcand remove_label wip

**Comment by chrisarcand** [comment]
@notchrisarcand remove_label asdfasdf

**Comment by notchrisarcand** [comment]
@chrisarcand Cannot remove the following label because they are not recognized: asdfasdf

**Comment by chrisarcand** [comment]
@notchrisarcand add_label alskdfjalskdjf

**Comment by notchrisarcand** [comment]
@chrisarcand Cannot apply the following label because they are not recognized: alskdfjalskdjf

**Comment by chrisarcand** [comment]
@notchrisarcand add_label wip

**Comment by chrisarcand** [comment]
@notchrisarcand add_label bug, asdfasdfalskdjfalskdjf

**Comment by notchrisarcand** [comment]
@chrisarcand Cannot apply the following label because they are not recognized: asdfasdfalskdjfalskdjf

**Comment by chrisarcand** [comment]
@notchrisarcand add label wontfix

**Comment by notchrisarcand** [comment]
@chrisarcand unrecognized command 'add', ignoring...

Accepted commands are: assign, set_milestone

**Comment by chrisarcand** [comment]
@notchrisarcand remove_labels wip, laskdjflaskjdf, wontfix

**Comment by notchrisarcand** [comment]
@chrisarcand Cannot remove the following label because they are not recognized: laskdjflaskjdf

**Comment by chrisarcand** [comment]
@notchrisarcand assign chrisarcand

**Comment by chrisarcand** [comment]
@notchrisarcand assign Freyishguyyyyyy

**Comment by notchrisarcand** [comment]
@chrisarcand 'Freyishguyyyyyy' is an invalid assignee, ignoring...

**Comment by chrisarcand** [comment]
@notchrisarcand set_milestone test

**Comment by chrisarcand** [comment]
@notchrisarcand set_milestone laksdjfals

**Comment by notchrisarcand** [comment]
@chrisarcand Milestone laksdjfals is not recognized, ignoring...

**Comment by chrisarcand** [comment]
@notchrisarcand dosmoething asdf

**Comment by notchrisarcand** [comment]
@chrisarcand unrecognized command 'dosmoething', ignoring...

Accepted commands are: add_label, assign, remove_label, rm_label, set_milestone

**Comment by jrafanie** [comment]
@notchrisarcand add_label wontfix

**Comment by jrafanie** [comment]
@notchrisarcand add_label wontfix

**Comment by notchrisarcand** [comment]
@miq-bot Only members of the matoska organization may use this command.

**Comment by notchrisarcand** [comment]
@Asdf Cannot apply the following label because they are not recognized: asdfasdf

**Comment by notchrisarcand** [comment]
@miq-bot Only members of the matoska organization may use this command.

**Comment by chrisarcand** [comment]
@notchrisarcand move_issue sandbox-2

**Comment by notchrisarcand** [comment]
@chrisarcand :x: `move_issue` failed. A pull request cannot be moved.

**Comment by chrisarcand** [comment]
@notchrisarcand move_issue miq_bot/sandbox-2

**Comment by notchrisarcand** [comment]
@chrisarcand :x: `move_issue` failed. A pull request cannot be moved, cannot move issue to repository outside of the matoska organization, and repository does not exist or is unreachable.

**Comment by chrisarcand** [comment]
@notchrisarcand move_issue sandbox

**Comment by notchrisarcand** [comment]
@chrisarcand :x: `move_issue` failed. Issue already exists on the 'sandbox' repository and a pull request cannot be moved.

**Comment by chrisarcand** [comment]
@notchrisarcand move_issue sandbox-2

**Comment by notchrisarcand** [comment]
@chrisarcand :x: `move_issue` failed. A pull request cannot be moved.

**Comment by chrisarcand** [comment]
@notchrisarcand move_issue ManageIQ/miq_bot

**Comment by notchrisarcand** [comment]
@chrisarcand :x: `move_issue` failed. A pull request cannot be moved and cannot move issue to repository outside of the matoska organization.

**Comment by chrisarcand** [comment]
@notchrisarcand close_issue

**Comment by chrisarcand** [comment]
@notchrisarcand close_issue

**Comment by chrisarcand** [comment]
@notchrisarcand close_issue

**Comment by notchrisarcand** [comment]
@miq-bot Only members of the matoska organization may use this command.

**Comment by chrisarcand** [comment]
@notchrisarcand add_label wip

**Comment by chrisarcand** [comment]
@notchrisarcand remove_label wip

**Comment by chrisarcand** [comment]
@notchrisarcand remove_label wip

**Comment by chrisarcand** [comment]
@notchrisarcand add_label wip

**Comment by chrisarcand** [comment]
@notchrisarcand remove_label wip

**Comment by chrisarcand** [comment]
@notchrisarcand close_issue

**Comment by chrisarcand** [comment]
@notchrisarcand close_issue

**Comment by chrisarcand** [comment]
@notchrisarcand add_label wip

**Comment by chrisarcand** [comment]
@notchrisarcand remove_label wip

**Comment by chrisarcand** [comment]
@notchrisarcand close_issue

**Comment by notchrisarcand** [comment]
@chrisarcand Only @notchrisarcand or a committer can close this pull request.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # sandbox
-A place for testing GitHub features/bots/etc. 
+A place for testib features/bots/etc.
```
Comment (chrisarcand) [inline README.md]: This commit is bad and you should feel bad.

**Pull request merged**